runtime/syntax/sml.vim

```diff
@@ -3,13 +3,16 @@
 " Filenames:    *.sml *.sig
 " Maintainers:  Markus Mottl            <markus.mottl@gmail.com>
 "               Fabrizio Zeno Cornelli  <zeno@filibusta.crema.unimi.it>
-" URL:          http://www.ocaml.info/vim/syntax/sml.vim
-" Last Change:  2006 Oct 23 - Fixed character highlighting bug (MM)
-"               2002 Jun 02 - Fixed small typo  (MM)
-"               2001 Nov 20 - Fixed small highlighting bug with modules (MM)
+" Last Change:  2019 Oct 01 - Only spell check strings & comments (Chuan Wei Foo)
+"               2015 Aug 31 - Fixed opening of modules (Ramana Kumar)
+"               2006 Oct 23 - Fixed character highlighting bug (MM)
 
 " quit when a syntax file was already loaded
 if exists("b:current_syntax")
+
+" Disable spell checking of syntax.
+syn spell notoplevel
```

Comment:
@mmottl I've just realized that this is on the wrong line. This should come after the `if exists(...) finish endif`.

In other words:

```diff
diff --git a/runtime/syntax/sml.vim b/runtime/syntax/sml.vim
index afff5304e..199b8f068 100644
--- a/runtime/syntax/sml.vim
+++ b/runtime/syntax/sml.vim
@@ -9,13 +9,12 @@
 
 " quit when a syntax file was already loaded
 if exists("b:current_syntax")
-
-" Disable spell checking of syntax.
-syn spell notoplevel
-
   finish
 endif
 
+" Disable spell checking of syntax.
+syn spell notoplevel
+
 " SML is case sensitive.
 syn case match
```